app/src/main/java/com/example/ncov19traking/utils/NumberFormatter.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.ncov19traking.utils
+
+import java.text.DecimalFormat
+
+class NumberFormatter
+
+fun formatLargeNumbers(value: Int): String {
+    val df = DecimalFormat("###,###,###")
+    return df.format(value)
+}
```

Comment:
so the intention was to create a utility class named `NumberFormatter` but here you have a class and a method that are not related to each other.

Reply:
Can you explain me a bit more? shall I use companion objects?

Reply:
fixed